### DISCUSSION TITLE
LTMEAT help

### DISCUSSION THREAD

**Discussion by sebastian418** [3kh0/ext-remover#639] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My district uses Lightspeed. So I got the Lightspeed ID and followed the instructions as you wrote. But after I finished clicking Bookmark C, no other pages loaded. I even followed the timeline on https://ltmeat.bypassi.com, but it didn't help me.

**Comment by deryilz** [comment, marked as answer]
ur too slow noob. use baby method which was made for babies like you https://ltmeat.bypassi.com/alt/1.txt

**Reply by sebastian418** [reply]
im not slow. i tried the baby method already that's why i'm asking for help you shaved coconut
go get a life
